fairmotion/tasks/pfnn/pfnn.py

```diff
@@ -0,0 +1,1753 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+"""
+This code is a Python reimplementation, authored by Jungdam Won, 
+of the C++ PFNN inference code from Daniel Holden's website
+https://theorangeduck.com/page/phase-functioned-neural-networks-character-control
+
+LICENSE
+
+This code and data is free for academic and non-commercial purposes but we 
+would ask that you please include the following citations in any published work 
+which uses this code or data.
+
+
+    @inproceedings{Holden:2017:PFNN,
+     author = {Holden, Daniel and Komura, Taku, and Saito, Jun},
+     title = {Phase-Functioned Neural Networks for Character Control},
+     booktitle = {SIGGRAPH 2017},
+     year = {2017},
+    }
+    
+    
+If you have any questions, or wish to enquire about commerical licenses, please 
+contact me at `contact@theorangeduck.com`.
```

Comment:
If you have any questions, or wish to enquire about commerical licenses, please 
contact me at `contact@theorangeduck.com`.

->> If you have any questions, or wish to enquire about commerical licenses, please 
contact to the original author at `contact@theorangeduck.com`.